browser_extension/css/style.css

```diff
@@ -1,74 +1,266 @@
+  font-family:
+    "HelveticaNeue-Light",
+    "Helvetica Neue Light",
+    "Helvetica Neue",
+    Helvetica,
+    Arial,
+    "Lucida Grande",
+    sans-serif;
+}
 
-/* CHAT AREA */
-.chat-area {
-  padding-bottom: 22px;
-  background-color: #45467949;
-  max-height: 540px;
-  max-width: 100%;
-  overflow-y: scroll;
-  display: flex;
-  flex-direction: column-reverse;
+html, body {
+  height: 100%;
+  margin: 0;
+  padding: 0;
 }
 
-#chatContentArea li {
+ul {
   list-style: none;
+  word-wrap: break-word;
+}
+
+
+.chatPage {
+  height: 100%;
+  background-color:#E0E0E0;
+  box-shadow: 2px 2px 5px rgba(0,0,0,0.5),
+            inset 5px 5px 8px rgba(0,0,0,0.5);
 }
+/* Pages */
 
-.chat-messages {
-  float: left;
+.pages {
+  height: 100%;
+  margin: 0;
   padding: 0;
+  width: 100%;
+}
+
+.page {
+  height: 100%;
+  position: absolute;
+  width: 100%;
+}
+
+.login.page .title {
+  font-size: 200%;
+}
+
+.login.page .usernameInput {
+  font-size: 100%;
+  letter-spacing: 3px;
+}
+
+.columnPair {
+  height: 94%;
+}
+
+.login.page .title, .login.page .usernameInput {
+  color: #000;
+  font-weight: 200;
+}
+
+/* Chat page */
+.messageArea {
+  height: 100%;
+}
+.chat.page {
+  background-color:#1e2833;
+}
+
+/* Font */
+.channelView {
+  background: #673AB7;
+  height: 100%;
 }
+.messages {
+  font-size: 100%;
+  color:#616161;
+}
+.fa-border {
+  color:black;
+}
+.themeGroup {
+  display: none;
+  background: #E0E0E0;
+  padding:2%;
+  position: absolute;
+          right:    7%;
+          bottom:   5%;
+          color:red;
+          border-radius: 25px;
+          border: 2px ;
 
-#chatContentArea {
-  text-align: left;
 }
 
-/* BOTTOM BAR */
-#bottomToolbar {
+.transition {
+  -webkit-transform:scale(1.3);
+  -moz-transform:scale(1.3);
+  transform:scale(1.3);
+  -o-transform:scale(1.3);
+}
+.theme1 {
+  color: #7E57C2;
+  font-size: 90%;
+}
+.theme2 {
+  color:#FF7043;
+  font-size: 90%;
+}
+.theme3 {
+ color: #66BB6A;
+ font-size: 90%;
+}
+.theme5 {
+  color: #90A4AE;
+  font-size: 90%;
+ }
+
+ .theme4 {
+  color: #B2EBF2;
+  font-size: 90%;
+ }
+.themeSettings {
   position: absolute;
-  left: 4px;
-  bottom: 5px;
+  left:    5%;
+  bottom:   5%;
+  font-size: 165%;
+  color: #E0E0E0;
+  border-radius: 25px;
+  border: 2px ;
 }
+.themeSettings:hover {
+  transform: scale(1.5) perspective(1px)
 
-#sendChatMessageBtn {}
+}
+.inputMessage {
+  font-size: 100%;
+  color: white;
+}
+
+.log {
+  color: gray;
+  font-size: 70%;
+  margin: 5px;
+  text-align: center;
+}
 
-#logoutBtn {}
\ No newline at end of file
+.innerA {
+  height: 98%;
+}
+
+/* Messages */
+
+.chatArea {
+  height: 100%;
+  padding-bottom: 60px;
+}
+
+.messages {
+  height: 100%;
+  margin: 0;
+  overflow-y: scroll;
+  padding: 10px 20px 10px 20px;
+}
+
+
+
+input {
+  color: #fff;
+}
+.message.typing .messageBody {
+  color: gray;
+}
+
+.username {
+  font-weight: 700;
+  overflow: hidden;
+  padding-right: 15px;
+  text-align: right;
+}
+
+.chaddon_headings {
```

Comment:
Property doesn't exist

Reply:
Property does exist and is used for the fonts